maquetado/setup/scss/titles.scss

```diff
@@ -0,0 +1,21 @@
+.title {
+  display: flex;
+  align-items: center;
+  font-family: 'Roboto', sans-serif;
```

Comment:
If you are going to use this font for the entire project, you can just add this prop on body element with a `class`.

`<body>` tag is special because it is the only one in the single project, you could use that one without a `class`